website/docs/docs/get-started/privatelink/about-privatelink.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "About PrivateLink"
+id: about-privatelink
+description: "Configuring PrivateLink for AWS"
+sidebar_label: "About PrivateLink"
+---
+
+
+PrivateLink enables you to connect your data platform hosted on AWS to a dbt Cloud multi-tenant environment over the AWS backbone using the PrivateLink technology. PrivateLink allows dbt Cloud customers to meet security and compliance controls as it restricts information shared between dbt Cloud and the data platform from reaching the public internet. This feature is supported in most regions across NA, Europe, and Asia, but [contact us](https://www.getdbt.com/contact/) if you have questions about availability. 
+
+To learn more, refer to the [AWS PrivateLink documentation](https://aws.amazon.com/privatelink/).
+
+dbt Cloud supports these data platforms that you can use the AWS PrivateLink feature with: 
+
+- Redshift
+- Snowflake
+- Databricks
+
+## Configure PrivateLink
+
+You need to contact the dbt Cloud support team to complete the integration. It is not a self-service feature at this time. Ensure that the subject contains "Multi-tenant PrivateLink" in some form. Please allow 1-2 business days for our teams to complete the setup on our side. 
```

Comment:
This seems like a prerequisite.

Reply:
They have to do some work on their side before this step. What might be a better way to frame it?